blueprints/modular/intro.yaml

```
@@ -59,6 +59,16 @@ form:
                   validate:
                     type: textarea
 
+                header.img_style:
+                  type: select
+                  label: Image Style
+                  description: Choose if the image should be a circle or have rounded corners.
+                  size: medium
+                  options:
+                    '': None
```

Let's rename this to make it a bit more obvious what no style means.
```suggestion
                    '': Square
```